test/hw11/test_task1.py

```diff
@@ -0,0 +1,89 @@
+import pytest
+
+from All_home_works.hw11.task1 import SimplifiedEnum
+
+
+class ColorsEnum(metaclass=SimplifiedEnum):
+    RED = "RED"
+    BLUE = "BLUE"
+    ORANGE = "ORANGE"
+    BLACK = "BLACK"
+
+
+@pytest.mark.parametrize(
+    "attr",
+    [
+        "RED",
+        "BLACK",
+        "BLUE",
+        "ORANGE",
+    ],
+)
+def test_simplified_enum_metaclass_works_fine_for_class(attr):
+    assert getattr(ColorsEnum, attr) == attr
+
+
+def test_simplified_enum_metaclass__if_attr_does_not_exist():
+    with pytest.raises(AttributeError, match="Does not exist:"):
+        _ = ColorsEnum.O
```

Comment:
`_ =` why do you need it?

Reply:
For me, it's more obvious to do this. And PyCharm doesn't blame me.